tools/encodings/GF Glyph Sets/GF-latin-pro_optional-glyphs.nam

```diff
@@ -0,0 +1,37 @@
+          at.case
+          hyphen.case
+          endash.case
+          emdash.case
+          periodcentered.case
+          bullet.case
+          slash.case
+          backslash.case
+          parenright.case
+          bracketleft.case
+          bracketright.case
+          braceleft.case
+          braceright.case
+          guilsinglleft.case
+          guilsinglright.case
+          guillemetleft.case
+          guillemetright.case
+          0x0030 zero.sinf
```

Comment:
all .sinf glyphs should be removed as per our guide on https://github.com/google/fonts/tree/master/tools/encodings/GF%202016%20Glyph%20Sets#gf-latin-pro-145-for-713-glyphs-total--17-optional-case